### PR TITLE
Label graph

### PR DESCRIPTION
**Pull request by sastels**
Add labels and title to graph, and use the theme font.

image

### CODE REVIEW

File: frontend/src/Stats.js [hunk 1]
```diff
@@ -11,28 +14,52 @@ import {
...
+                  data={this.props.data}
+                  margin={{
+                    top: 0,
+                    right: 0,
+                    left: -20,
+                    bottom: 0,
+                  }}
+                >
+                  <CartesianGrid strokeDasharray="2 2" />
+                  <XAxis fontFamily={theme.fontSans} dataKey="date" />
```
Comment (sastels) [inline frontend/src/Stats.js]: Using the `ThemeProvider` didn't seem to work here, unless I was doing it the wrong way.

File: frontend/src/components/header/index.js [hunk 2]
```diff
@@ -11,6 +18,7 @@ export const Header = styled(tag)`
   ${space};
   ${color};
   ${fontWeight};
+  ${textAlign};
```
Comment (sastels) [inline frontend/src/components/header/index.js]: wanted to be able to center a header

File: frontend/src/Stats.js [hunk 3]
```diff
@@ -11,28 +14,52 @@ import {
...
+                  <YAxis
+                    dataKey="total"
+                    fontFamily={theme.fontSans}
+                    label={{
+                      value: i18n._(t`Reports per day`),
```
Comment (sastels) [inline frontend/src/Stats.js]: Couldn't just use `<Trans>` here, instead used the approach suggested in https://lingui.js.org/tutorials/react-patterns.html#element-attributes-and-string-only-translations

File: frontend/src/Stats.js (hunk 1 above)
Comment (anikbrazeau) [reply frontend/src/Stats.js]: Can we make these changes in French too?

Title EN: Reports similar to yours
Title FR: Rapports semblables au vôtre

Y-axis EN: Reports per day
Y-axis FR: Rapports par jour

Comment (sastels) [reply frontend/src/Stats.js]: Right now the French (just from DeepL) is:
```
"Reports per day": "Rapports par jour",
"Reports similar to yours": "Des rapports similaires aux vôtres",
```
Can change the French title as you suggest 👍

Comment (sastels) [reply frontend/src/Stats.js]: image

**Comment by keithiancolbourne** [comment]
Looks good! Are any tests needed?

**Comment by sastels** [comment]
> Looks good! Are any tests needed?

I figured no. Just adding labels to the graph 🤷‍♂️

**Comment by keithiancolbourne** [comment]
> > Looks good! Are any tests needed?
> 
> I figured no. Just adding labels to the graph 🤷‍♂️

Yeah, same. Only thing comes to mind is an accessibility test. i.e. checking to see if there's actually a label kinda thing

**Pull request merged**